src/group_theory/free_group.lean

```diff
@@ -820,6 +833,14 @@ by rintros ⟨L⟩; exact reduce.self
 lemma to_word.inj : ∀(x y : free_group α), to_word x = to_word y → x = y :=
 by rintros ⟨L₁⟩ ⟨L₂⟩; exact reduce.exact
 
+theorem to_word.le_length : (mk L₁).to_word.length ≤ L₁.length := 
```

Comment:
You define LHS to be `norm (mk L1)` later. I think that the lemma should be reformulated as `norm_mk_le` after the definition.

Reply:
Yes this is a good idea and makes the meaning clearer.   8e2f8067019434aa09e691ddd5d2f68e81d11749 does this.